analysis/deep2_r23_o32/calibration_plots.py

```diff
@@ -84,13 +86,15 @@ def lac_gpc_plots(fitspath, fitspath_ini, dataset, raw=False,
     # DEEP2 Derived
     er_R23 = derived_deep_tab['R23'].data
     er_O32 = derived_deep_tab['O32'].data
+    der_ID = derived_deep_tab['ID'].data
```

Comment:
```suggestion
    der_ID = derived_deep_tab['ID'].data
```

We really need to pick better names for these variables. I have no idea what `er` is vs `der`.

Reply:
Changed this in my most recent commit